crates/bevy_pbr/src/render/mesh_preprocess_types.wgsl

```diff
@@ -0,0 +1,19 @@
+// Types needed for GPU mesh uniform building.
+
+#define_import_path bevy_pbr::mesh_preprocess_types
+
+// The `wgpu` indirect parameters structure. This is a union of two structures.
+// For more information, see the corresponding comment in
+// `gpu_preprocessing.rs`.
+struct IndirectParameters {
+    // `vertex_count` or `index_count`.
+    vertex_count_or_index_count: u32,
+    // `instance_count` in both structures.
+    instance_count: atomic<u32>,
+    // `first_vertex` in both structures.
+    first_vertex: u32,
```

Comment:
This comment does not agree with the name or comments on the Rust struct, there it's `first_vertex_or_first_index`.

Reply:
I changed it to `first_vertex_or_first_index`.